cupy/core/fusion.pyx

```diff
@@ -162,7 +162,7 @@ class _FusionOp(object):
 
     def __repr__(self):
         return '<_FusionOp #{}, {} types=[{}]>'.format(
-            self.index, self.submodule.name, ', '.join(self.dtypes))
+            self.index, self.submodule.name, ', '.join(map(str, self.dtypes)))
```

Comment:
Is this fix correct?

Reply:
The original code raises `TypeError: sequence item 0: expected str instance, numpy.dtype found`.